src/main/java/net/dv8tion/jda/core/entities/User.java

```diff
@@ -111,6 +113,14 @@
     RestAction<PrivateChannel> openPrivateChannel();
 
     /**
+     * Finds and collects all {@link net.dv8tion.jda.core.entities.Guild Guild} instances that contain this {@link net.dv8tion.jda.core.entities.User User} within the current {@link net.dv8tion.jda.core.JDA JDA} instance.<br>
```

Comment:
Technically, `<br>` should be on the next line In this case however, there is no line that is part of this paragraph after this line so there actually shouldn't even be a `<br>` as the following `<p>` handles the paragraph separation.